snap/snapcraft.yaml

```diff
@@ -0,0 +1,99 @@
+name: edgex-device-rest
+base: core18
+type: app
+version: "replace-me"
+version-script: |
+  echo $(cat VERSION)-$(date +%Y%m%d)+$(git rev-parse --short HEAD)
+license: Apache-2.0
+title: EdgeX REST Device Service
+summary: EdgeX device service for REST protocol
+description: |
+  This device service provides easy way for 3'rd party applications, 
+  such as Point of Sale, CV Analytics, etc., to push data into EdgeX 
+  via the REST protocol.
+
+# delhi: 0, edinburgh: 1, fuji: 2, geneva: 3
+epoch: 3
+
+# TODO: add armhf here when that's supported
+architectures:
+  - build-on: arm64
+  - build-on: amd64
+
+grade: stable
+confinement: strict
+
+apps:
+  device-rest-go:
+    adapter: none
+    command: bin/device-rest-go $CONFIG_PRO_ARG $CONF_ARG $REGISTRY_ARG
+    daemon: simple
+    environment:
+      CONFIG_PRO_ARG: "--cp=consul://localhost:8500"
+      CONF_ARG: "--confdir=$SNAP_DATA/config/device-rest-go/res"
+      REGISTRY_ARG: "--registry"
+      DEVICE_PROFILESDIR: "$SNAP_DATA/config/device-rest-go/res"
```

Comment:
can remove this, since not used elsewhere

Reply:
The DEVICE_PROFILESDIR is the env variable override used by the service so it can find the profiles directory.

Reply:
It is used by the device-sdk/bootstrap